src/components/Garden/ModalFlows/SupportProposal/SupportProposalScreens.js

```diff
@@ -1,16 +1,66 @@
 import React, { useMemo, useState, useCallback } from 'react'
+import { Button } from '@1hive/1hive-ui'
+import { useHistory } from 'react-router'
+import { buildGardenPath } from '@utils/routing-utils'
 import ModalFlowBase from '../ModalFlowBase'
 import ChangeSupport from './ChangeSupport'
 import SupportProposal from './SupportProposal'
 
 import useActions from '@hooks/useActions'
 
 function SupportProposalScreens({ proposal, mode }) {
+  const history = useHistory()
   const [transactions, setTransactions] = useState([])
   const { convictionActions } = useActions()
 
   const { id: proposalId } = proposal
 
+  const handleViewProposals = useCallback(() => {
+    const path = '/profile'
+    history.push(path)
+  }, [history])
+
+  const handleViewHome = useCallback(() => {
+    const path = buildGardenPath(history.location, 'home')
+    history.push(path)
+  }, [history])
+
+  const renderOnCompleteActions = useCallback(() => {
```

Comment:
I am not convinced with this modal screen yet. I know we already discuss that it will be a temporal improvement until we support something like notify and just close the modal with a toast of the transaction displayed.